tests/unit/editors/dateEditor/tests.html

```diff
@@ -68,7 +68,7 @@
 						value: '01/09/50',
 						dateInputFormat: "dd/MM/yyyy",
 						dateDisplayFormat: "dd/MM/yyyy",
-						centuryThreshold: 101
+						//centuryThreshold: 101
```

Comment:
There is a new test created in order to test the centuryThreshold option, and it is removed here as it is no longer relevant for the tests using this editor instance.